shap/benchmark/methods.py

```diff
@@ -88,6 +90,16 @@ def tree_gain(model, data):
     """
     return other.TreeGainExplainer(model).attributions
 
+def permutation_shap_1(model, data):
+    """ Permutation Explainer
+    """
+    return lambda X: PermutationExplainer(model.predict, data).shap_values(X, npermutations=1)
```

Comment:
hmm why does this need to return a lambda?
Also, why is the name _1 here and _10 below, is that corresponding to the npermutations?  Maybe add a comment in the method.

Reply:
Will remove this because it is for testing and no longer valid.